From f5cfc9aad77f102458e1b368febb9d81b63a1fb9 Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Wed, 5 Jul 2017 19:20:04 +0300
Subject: [PATCH 3647/5242] MLK15034-4: ARM: cpuidle imx7d: Declare longer
 exit_latency/target_residency

commit  aff47ee4d43cf24ceeeb055f47b49bc1f8036575 from
https://source.codeaurora.org/external/imx/linux-imx.git

Low power idle exit latency is much longer than declared, in the
milisecond range.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-imx/cpuidle-imx7d.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-imx/cpuidle-imx7d.c b/arch/arm/mach-imx/cpuidle-imx7d.c
index 641cdca..40dcdce 100644
--- a/arch/arm/mach-imx/cpuidle-imx7d.c
+++ b/arch/arm/mach-imx/cpuidle-imx7d.c
@@ -207,8 +207,8 @@ static int imx7d_enter_low_power_idle(struct cpuidle_device *dev,
 		},
 		/* LOW POWER IDLE */
 		{
-			.exit_latency = 500,
-			.target_residency = 800,
+			.exit_latency = 10000,
+			.target_residency = 20000,
 			.flags = CPUIDLE_FLAG_TIMER_STOP,
 			.enter = imx7d_enter_low_power_idle,
 			.name = "LOW-POWER-IDLE",
-- 
1.7.9.5

